### ISSUE TITLE
notify - allow heading aria-level to be set via config.json

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#2486]
One of our clients requires that the aria-level for the Notify title element should be set to 3 (and not 1 like it is by default), saying that 'each page should only have one level 1 heading'.

Personally I'm not sure I agree with what they're saying - but not to the point I'm going to argue, if that's what they want then I'm happy to make that change... but it would be easier to do so (in fact the only way to do so for AAT projects) if it could be configurable via a property in `_accessibility._ariaLevels` in config.json
